tests/integration/test_bayesian_optimization.py

```diff
@@ -83,13 +87,15 @@
                 num_query_points=2,
             ),
         ),
-        (15, TrustRegion()),
         (10, DiscreteThompsonSampling(500, 3)),
         (10, DiscreteThompsonSampling(500, 3, num_fourier_features=1000)),
     ],
 )
+@pytest.mark.parametrize("trust_region", [continuous_trust_region(), None])
```

Comment:
this doubles the number of integration tests in this file. any preference of which tests to apply trust region to?

Reply:
maybe one EGO based and one TS ?

Reply:
done